assignments/callbacks.js

```diff
@@ -24,25 +24,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
```

Comment:
You did a good job with the callbacks but I encourage you to work on your es6 syntax. It will make learning jsx much easier and seems to be most common practice.